.ci/deps.sh

```diff
@@ -58,7 +58,7 @@ if [ ! -e ~/phpmd/phpmd ]; then
 fi
 
 # astyle installation
-wget "https://downloads.sourceforge.net/project/astyle/astyle/astyle%203.0.1/astyle_3.0.1_linux.tar.gz?r=&ts=1499017588&use_mirror=excellmedia" -O ~/astyle.tar.gz
+wget "https://downloads.sourceforge.net/project/astyle/astyle/astyle%203.0.1/astyle_3.0.1_linux.tar.gz?r=&ts=1499017588&use_mirror=excellmedia" -O ~/astyle.tar.gz --no-check-certificate
```

Comment:
why this?
surely that would be a separate commit.

Reply:
This had fixed an issue with the certificates
Will get a link to the error and see if there is a better solution